etna/recordseries/models.py

```diff
@@ -0,0 +1,48 @@
+from django.contrib.contenttypes.fields import GenericRelation
+from django.db import models
+
+from wagtail.admin.panels import FieldPanel
+from wagtail.images import get_image_model_string
+from wagtail.models import LockableMixin, RevisionMixin
+from wagtail.search import index
+
+
+class RecordSeries(LockableMixin, RevisionMixin, index.Indexed, models.Model):
+    admin_name = models.CharField(
+        max_length=128,
+        help_text="The name of the series as it should be displayed in the CMS site.",
+    )
+    title = models.CharField(max_length=100)
+    introduction = models.TextField(max_length=512)
+    image = models.ForeignKey(
+        get_image_model_string(), on_delete=models.SET_NULL, null=True, blank=False
+    )
+    show_only_digitised_records = models.BooleanField(default=True)
+
+    # TODO: Once the real integration with the CIIM search is built,
+    #       we should add a field that will allow identify the series
+    #       in the CIIM search and on record detail pages.
+
+    revisions = GenericRelation("wagtailcore.Revision", related_query_name="advert")
```

Comment:
What does this do for us? I've not seen it used before

Reply:
You can track revisions of content on snippets, but it's an opt-in behaviour. I thought it may be nice to enable that to be able to see what's been changed and be able to rollback to the previous version.

See: https://docs.wagtail.org/en/stable/topics/snippets/features.html#saving-revisions-of-snippets